### DISCUSSION TITLE
Package dependancy for Hirsute

### DISCUSSION THREAD

**Discussion by Sean-StarLabs** [fwupd/fwupd#3341] [Q&A] [answered]
Debian package doesn't work when built for Hirsute, has the error `/usr/libexec/fwupd/efi/fwupdx64.efi cannot be found`.

Seems to be dependency related, as`libtss2-esys0` doesn't exist and has been replaced with `libtss2-esys-3.0.2-0`

Not sure how to submit a PR, as changing the package name in the `dependancies.xml` would break more version than it would fix.

**Comment by superm1** [comment, marked as answer]
It's intentionally split, but the packages aren't landed in Ubuntu or Debian yet.   Build the package for the EFI binary from here: http://github.com/fwupd/fwupd-efi

**Reply by Sean-StarLabs** [reply]
Sorry, missed that!

As for the dependency, is there a way to have it that'll work on both? Right now, can't build from source on Hirsute (can changing things but that'll break focal).

**Reply by superm1** [reply]
Yeah You can install same package on both focal and hirsute like this.

**Reply by Sean-StarLabs** [reply]
Like what?

**Reply by superm1** [reply]
You can build fwupd-efi for focal or hirsute, and then install them before you build.  And they'll cover that dependency then.

**Reply by Sean-StarLabs** [reply]
Gotcha! Thanks 👍
